kostyor/db/api.py

```diff
@@ -133,6 +139,12 @@ def create_service(name, host_id, version):
             'version': new_service.version}
 
 
+def get_services_by_host(host_id):
+    services = db_session.query(models.Service).filter_by(
+        host_id=host_id).all()
```

Comment:
minor: there's no need to produce a new list by calling `.all()`. query supports iterator protocol, so you can just iterate over its results in the list comprehension below.